zubbi/templates/includes/role-example.html

```diff
@@ -51,5 +54,9 @@
 
    This variable must be a dictionary. As it does not provide any default
    value and thus, must be set to use this role.
+
+<span class="p">..</span> <span class="ow">supported_os</span><span class="p">::</span> Linux, Windows
+
+<span class="p">..</span> <span class="ow">reusable</span><span class="p">::</span> True
```

Comment:
I would change this block to the following:

```
<span class="p">..</span> <span class="ow">supported_os</span><span class="p">::</span> Linux, Windows

   List of supported operating systems, separated by comma. Currently supported
   values are <b>Linux</b> and <b>Windows</b>.

<span class="p">..</span> <span class="ow">reusable</span><span class="p">::</span> True

   Set this to <b>True</b> if the role should be highlighted in Zubbi
```

Reply:
I was also thinking about putting the description under the directives, but I am afraid the reader may misunderstand it, as there are descriptions required for this directives?

Reply:
I think the same applies for the current directives. AFAIK you don't need a description for every one of them. But since this is a HOW TO guide I think it's fine to put them there. There is also the job example below where those directives don't have any description.

Reply:
Also fine with me. I will update.

Reply:
Another idea would be to write this as a comment (starting with `#`) above the directive.